local_tf/codegen.py

```diff
@@ -0,0 +1,63 @@
+# depends on grpcio-tools
+
+from grpc.tools import protoc
+
+import sys
+import os
+import shutil
+import tempfile
+
+tensorflow_root         = sys.argv[1]
+tensorflow_serving_root = sys.argv[2]
```

Comment:
Those needs to be the git repos on a developer's system ; not the installation path. The codegen.py is supposed to be run rarely, to be in sync with the Tensorflow Serving APIs requirements, and then checked-in.